spec/controllers/taxons_controller_spec.rb

```diff
@@ -1,9 +1,7 @@
-require "test_helper"
 require_relative "../../test/support/coronavirus_helper"
 
-describe TaxonsController do
+RSpec.describe TaxonsController do
   include SearchApiHelpers
-  include GovukAbTesting::MinitestHelpers
   include TaxonHelpers
 
   describe "GET show" do
```

Comment:
It doesn't seem to have converted this file (`assert_response` etc)

Reply:
hmm, I think that might be one of my classic speed rebase errors. will fix sorry!